CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Remove trailing slash from endpoint url by [@Procta].
 - Added local cache to getResolutions by [@jpastoor].
 - Renamed Api::api() parameter $return_as_json to $return_as_array by [@jpastoor].
+- ignore_errors flag added to HTTP context by [@betterphp].
```

Comment:
1. this should be added to `Fixed` section
2. this a fix to case, when error text not being available as API call result and should be written as Error details from failed API calls were not available back from `Api::api` method call.
3. for your nickname to be highlighted on changelog page you also need to add it in mapping at the bottom of this file